cmd/hubagent/workload/setup.go

```diff
@@ -323,7 +323,7 @@ func SetupControllers(ctx context.Context, wg *sync.WaitGroup, mgr ctrl.Manager,
 		if err := (&schedulercrbwatcher.Reconciler{
 			Client:             mgr.GetClient(),
 			SchedulerWorkQueue: defaultSchedulingQueue,
-		}).SetupWithManager(mgr); err != nil {
+		}).SetupWithManagerForClusterResourceBinding(mgr); err != nil {
```

Comment:
I think it's better to add the namespaced resource reconciler instance with these PRs. Otherwise, we would need a separate one to collect all the reconcilers that need a separate instance and that would require extra time and can be error prone.

Reply:
we'll init all the RP controllers in one PR along with PR e2e tests.